packages/showcases/css/stories/components/text-input/examples/error.html

```diff
@@ -1,10 +1,13 @@
 <div class="block">
   <label class="vtmn-text-input_label" for="my-text-input-error">Label</label>
-  <input
-    type="text"
-    class="vtmn-text-input vtmn-text-input--error"
-    id="my-text-input-error"
-    placeholder="Placeholder Text"
-  />
-  <p class="vtmn-text-input_error-text">Error message goes here</p>
+  <div class="vtmn-text-input_container">
+    <input
+      type="text"
+      class="vtmn-text-input vtmn-text-input--error"
+      id="my-text-input-error"
+      placeholder="Placeholder Text"
+    />
+    <span class="vtmx-search-line"></span>
```

Comment:
Maybe we could use a more readable naming, like vtmn-text-input_icon ?

Reply:
Thanks Gabin but for this span class, I'm using the icon name of the font to display it.

Reply:
But I could add a more relevant class name too indeed